edgePy/DGEList.py

```diff
@@ -272,37 +306,120 @@ def library_size(self) -> np.array:
         """
         return np.sum(self.counts, 0)
 
-    def cpm(self, log: bool = False, prior_count: float = PRIOR_COUNT) -> None:
-        """Return the DGEList normalized to read counts per million."""
-        self.counts = 1e6 * self.counts / np.sum(self.counts, axis=0)
-        if log:
-            self.counts[self.counts == 0] = prior_count
-            self.counts = np.log(self.counts)
+    def log_transform(self, counts, prior_count):
```

Comment:
These changes are introduced by the tpm branch (just to let you know)

Reply:
Yes!  I was looking forward to them - thanks for implementing!